From a11985b705526cd628c0cad46382c76cb13febe8 Mon Sep 17 00:00:00 2001
From: Anil Patel <anil.patel@softwebsolutions.com>
Date: Thu, 10 Aug 2023 12:53:25 +0530
Subject: [PATCH] MTD: SPI-NOR: Added support for at25sf321 nor flash

Signed-off-by: Anil Patel <anil.patel@softwebsolutions.com>
---
 drivers/mtd/spi-nor/atmel.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/mtd/spi-nor/atmel.c b/drivers/mtd/spi-nor/atmel.c
index e9d7872882c1..d7c4a8aeb78d 100644
--- a/drivers/mtd/spi-nor/atmel.c
+++ b/drivers/mtd/spi-nor/atmel.c
@@ -66,6 +66,7 @@ static const struct flash_info atmel_parts[] = {
 	{ "at25df321",  INFO(0x1f4700, 0, 64 * 1024,  64, SECT_4K | SPI_NOR_HAS_LOCK) },
 	{ "at25df321a", INFO(0x1f4701, 0, 64 * 1024,  64, SECT_4K | SPI_NOR_HAS_LOCK) },
 	{ "at25df641",  INFO(0x1f4800, 0, 64 * 1024, 128, SECT_4K | SPI_NOR_HAS_LOCK) },
+	{ "at25sf321", INFO(0x1f8701, 0, 64 * 1024, 64, SECT_4K | SPI_NOR_DUAL_READ | SPI_NOR_QUAD_READ) },
 
 	/* Dialog */
 	{ "at25ql128a",	 INFO(0x1f4218, 0, 64 * 1024,  256, SECT_4K | SPI_NOR_QUAD_READ) },
-- 
2.17.1

